Controller/ResourceController.php

```diff
@@ -123,17 +123,39 @@ protected function findOrThrowNotFound(ResourceInterface $resource, $id)
     protected function getCollectionData(ResourceInterface $resource, Request $request)
     {
         $page = (int) $request->get('page', 1);
-
+        $order = [];
         $filters = [];
-        foreach ($resource->getFilters() as $resourceFilter) {
+        $resourceFilters = $resource->getFilters();
+        $resourceOrders = $resource->getOrder();
+
+        foreach ($resourceFilters as $resourceFilter) {
             if (null !== $value = $request->get($resourceFilter['name'])) {
                 $resourceFilter['value'] = $value;
                 $filters[] = $resourceFilter;
             }
         }
 
+        // Add order filters
+        $requestOrderFilters = (null !== $request->get('order'))?
```

Comment:
Will fail is an user write an URL like mapge?order=test.

For readability and performances:
```php
$requestOrder = $request->get('order');
$requestOrder = is_array($order) ? $requestOrder : null;
```